src/overlays/actors/ovl_Bg_Mizu_Movebg/z_bg_mizu_movebg.c

```diff
@@ -82,84 +92,85 @@ void BgMizuMovebg_Init(Actor* thisx, PlayState* play) {
     s32 type;
     s32 waypointId;
     WaterBox* waterBoxes = play->colCtx.colHeader->waterBoxes;
-    f32 temp;
+    f32 targetPosY;
     CollisionHeader* colHeader = NULL;
-    Vec3f sp48;
+    Vec3f offsetPos;
 
     Actor_ProcessInitChain(thisx, sInitChain);
     ((BgMizuMovebg*)thisx)->homeY = thisx->world.pos.y;
-    ((BgMizuMovebg*)thisx)->dlist = D_8089EB50[MOVEBG_TYPE(thisx->params)];
```

Comment:
it looks like this change was not even yours, but we like to have "display list" as two words, so the shortening of it for the instance variable should be `dList`

This is something that needs cleaning up across the codebase in general, but figured while youre here